articles/_includes/_new_app.md

```diff
@@ -1,9 +1,11 @@
 ## Configure Auth0
 ### Get Your Application Keys
 
-When you signed up for Auth0, a new application was created for you, or you could have created a new one.
+When you signed up for Auth0, a new application was created for you, or you could have created a new one. You will need some details about that application to communicate with Auth0. You can get these details from the [Application Settings](${manage_url}/#/applications) section in the Auth0 dashboard.
 
-You will need some details about that application to communicate with Auth0. You can get these details from the [Application Settings](${manage_url}/#/applications) section in the Auth0 dashboard.
+<% if(typeof hideDashboardScreenshot === 'undefined' || hideDashboardScreenshot !== true) { %>
+![App Dashboard](/media/articles/dashboard/client_settings.png)
+<% } %>
```

Comment:
It makes more sense to have the screenshot of the Dashboard right below where it's mentioned. It also provides a little break from the repetition of all those `You`s.